app/navigation/Navigation.scala

```diff
@@ -45,6 +45,12 @@ class Navigation extends Navigator {
           case true  => controllers.routes.UnloadingCommentsController.onPageLoad(ua.id, NormalMode)
           case false => controllers.routes.CheckYourAnswersController.onPageLoad(ua.id)
         }
+    case DoYouHaveAnythingElseToReportYesNoPage =>
```

Comment:
I think we can ignore navigation for now, there should be a separate ticket for that

Reply:
Ok done